src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -173,7 +174,7 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
         },
         refreshClusters: () => this.refreshClusters()
       }, [
-        workspace && h(WrappedComponent, {
+        workspace && h(toClass(WrappedComponent), {
```

Comment:
Doing this in the `render` path will create a new component every time, which will probably cause unwanted re-mounting. We should do this once at the top of `wrapWorkspace`